include/kodlab_mjbots_sdk/behavior_manager.h

```diff
@@ -266,26 +289,38 @@ class BehaviorManager
    * @param behavior_idx index of behavior
    * @return name of behavior at index `behavior_idx` in `behaviors_`
    */
-  [[nodiscard]] std::string get_behavior_name(int behavior_idx) const
-  {
+  [[nodiscard]] std::string get_behavior_name(int behavior_idx) const {
     return behaviors_[behavior_idx]->get_name();
   }
 
+  /**
+   * @brief Lookup for behavior index by name
+   * @param name behavior name
+   * @return index of behavior if behavior is in the behavior list, -1 otherwise
+   */
+  [[nodiscard]] int get_behavior_index(const std::string &name) {
+    auto it = name_to_idx_map_.find(name);
+    if (it != name_to_idx_map_.end()) {
+      return it->second;
+    } else {
+      LOG_WARN("%s is not in behavior list.", name.c_str());
+      return -1;
```

Comment:
std::optional. Other option would be to return the iterator, and have the use check if its equal to the end, but the std::optional is a clearer design

Reply:
Agreed `std::optional` is the way.  Added in 6bc58c22c7bfe4d156e3962308c3fbf73fc639c6